Build failure after project clean

This happens pretty regularly and it usually takes about 3-5 cleans and builds before it actually builds successfully. 
<!-- Please note which version of the Mapbox Navigation SDK you are using. -->

**Mapbox Navigation SDK version:**
MapboxNavigation (0.31.0)

**Xcode 10.2**

**Linker Command**
```
Ld /Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Products/Debug-iphonesimulator/MapboxNavigation/MapboxNavigation.framework/MapboxNavigation normal x86_64 (in target: MapboxNavigation)
    cd /Users/Matt/Development/Applied/DAP-iOS/Pods
    export IPHONEOS_DEPLOYMENT_TARGET=9.0
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -arch x86_64 -dynamiclib -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator12.2.sdk -L/Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Products/Debug-iphonesimulator/MapboxNavigation -F/Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Products/Debug-iphonesimulator/MapboxNavigation -F/Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Products/Debug-iphonesimulator/MapboxCoreNavigation -F/Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Products/Debug-iphonesimulator/MapboxDirections.swift -F/Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Products/Debug-iphonesimulator/MapboxMobileEvents -F/Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Products/Debug-iphonesimulator/MapboxSpeech -F/Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Products/Debug-iphonesimulator/Polyline -F/Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Products/Debug-iphonesimulator/Solar -F/Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Products/Debug-iphonesimulator/Turf -F/Users/Matt/Development/Applied/DAP-iOS/Pods/Mapbox-iOS-SDK/dynamic -F/Users/Matt/Development/Applied/DAP-iOS/Pods/MapboxNavigationNative -filelist /Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/MapboxNavigation.build/Objects-normal/x86_64/MapboxNavigation.LinkFileList -install_name @rpath/MapboxNavigation.framework/MapboxNavigation -Xlinker -rpath -Xlinker /usr/lib/swift -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -mios-simulator-version-min=9.0 -dead_strip -Xlinker -object_path_lto -Xlinker /Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/MapboxNavigation.build/Objects-normal/x86_64/MapboxNavigation_lto.o -Xlinker -export_dynamic -Xlinker -no_deduplicate -Xlinker -objc_abi_version -Xlinker 2 -fobjc-arc -fobjc-link-runtime -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator -Xlinker -add_ast_path -Xlinker /Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/MapboxNavigation.build/Objects-normal/x86_64/MapboxNavigation.swiftmodule -framework Foundation -framework MapboxCoreNavigation -framework MapboxSpeech -framework Solar -compatibility_version 1 -current_version 1 -Xlinker -dependency_info -Xlinker /Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/MapboxNavigation.build/Objects-normal/x86_64/MapboxNavigation_dependency_info.dat -o /Users/Matt/Library/Developer/Xcode/DerivedData/D.A.P-gfqbwrnlyoknnyffsexhiohgzify/Build/Products/Debug-iphonesimulator/MapboxNavigation/MapboxNavigation.framework/MapboxNavigation
```

**Build error:** 
```
Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_MMEEventsManager", referenced from:
      objc-class-ref in CarPlayManager.o
      objc-class-ref in RouteMapViewController.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,7 @@
 * This SDK can no longer be used in applications written in pure Objective-C. If you need to use this SDK’s public API from Objective-C code, you will need to implement a wrapper in Swift that bridges the subset of the API you need from Swift to Objective-C. ([#2230](https://github.com/mapbox/mapbox-navigation-ios/pull/2230))
 * Added a new dependency on MapboxAccounts so that this SDK’s usage of Mapbox APIs is [billed](https://www.mapbox.com/pricing/) together based on [monthly active users](https://docs.mapbox.com/help/glossary/monthly-active-users/) rather than individually by HTTP request. If you use Carthage to install this SDK, remember to add MapboxAccounts.framework to the “Frameworks, Libraries, and Embedded Content” section and the input and output file lists of Carthage’s Run Script build phase. ([#2151](https://github.com/mapbox/mapbox-navigation-ios/pull/2151))
 * Upgraded to [Mapbox Maps SDK for iOS v5.6._x_](https://github.com/mapbox/mapbox-gl-native-ios/releases/tag/ios-v5.6.0). ([#2302](https://github.com/mapbox/mapbox-navigation-ios/pull/2302))
+* Made dependency on MapboxMobileEvents explicit in the podspec. [#2094](https://github.com/mapbox/mapbox-navigation-ios/issues/2094)
 
 ### Top and bottom banners
 * Removed `BottomBannerViewController(delegate:)` in favor of `BottomBannerViewController()` and the `BottomBannerViewController.delegate` property’s setter. ([#2297](https://github.com/mapbox/mapbox-navigation-ios/pull/2297))

--- a/MapboxNavigation.podspec
+++ b/MapboxNavigation.podspec
@@ -47,6 +47,7 @@ Pod::Spec.new do |s|
   s.dependency "Mapbox-iOS-SDK", "~> 5.6"
   s.dependency "Solar", "~> 2.1"
   s.dependency "MapboxSpeech", "~> 0.3.0"
+  s.dependency "MapboxMobileEvents", "~> 0.10.2"
 
   s.swift_version = "5.0"
 

